sdk/monitor/azure-monitor-opentelemetry-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/LogDataMapper.java

```diff
@@ -28,6 +30,8 @@ public class LogDataMapper {
 
     private static final ClientLogger logger = new ClientLogger(LogDataMapper.class);
 
+    private static final Set<String> EXCLUDE_LOGGER_NAMES = new HashSet<>(1);
```

Comment:
probably worth initializing from a method to ensure [safe publication](https://shipilev.net/blog/2014/safe-public-construction/#_safe_publication), e.g.

```suggestion
    private static final Set<String> EXCLUDE_LOGGER_NAMES = getExcludeLoggerNames();
```